Urban-Fun/View Controllers/TimelineViewController.m

```diff
@@ -98,6 +116,12 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     return cell;
 }
 
+- (void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)cell forRowAtIndexPath:(NSIndexPath *)indexPath{
+    if(indexPath.row + 2 == [self.arrayOfActivities count]){
```

Comment:
Let's say you only have one activity in your feed, is this going to work?
You can potentially improve this code by using `UIScrollViewDelegate` instead, I found this [tutorial](https://guides.codepath.com/ios/Simulated-Infinite-Scrolling) that can help you with that

Reply:
done